tst/standard/TensorProductMatrixGroups.tst

```diff
@@ -24,17 +30,18 @@ gap> TestTensorProductStabilizerInSU := function(args)
 >   G := TensorProductStabilizerInSU(d1, d2, q);
 >   hasSize := HasSize(G);
 >   RECOG.TestGroup(G, false, Size(G));
->   return IsSubset(SU(d1 * d2, q), GeneratorsOfGroup(G))
->          and DefaultFieldOfMatrixGroup(G) = GF(q ^ 2)
->          and hasSize;
+>   Assert(0, IsSubset(SU(d1 * d2, q), GeneratorsOfGroup(G)));
+>   Assert(0, DefaultFieldOfMatrixGroup(G) = GF(q ^ 2));
+>   Assert(0, hasSize);
 > end;;
+gap> TestTensorProductStabilizerInSU([2, 3, 2]);
+gap> TestTensorProductStabilizerInSU([2, 3, 3]);
+gap> TestTensorProductStabilizerInSU([2, 3, 4]);
 #@if IsBound(CLASSICAL_MAXIMALS_RUN_BROKEN_TESTS)
-gap> testsTensorProductStabilizerInSU := [[2, 3, 2], [2, 3, 3], [2, 3, 4], [2, 4, 3]];;
-#@else
-gap> testsTensorProductStabilizerInSU := [[2, 3, 2], [2, 3, 3]];;
+gap> TestTensorProductStabilizerInSU([2, 4, 3]); # FIXME: see https://github.com/gap-packages/recog/issues/302
```

Comment:
For reference / linking issues: https://github.com/gap-packages/recog/issues/302